components/automate-chef-io/README.md

```diff
@@ -81,11 +88,314 @@ This will create the file and fill in the necessary frontmatter automatically.
+
+### API File Index
+
+* **API Proto Files**: automate/components/automate-gateway/api/
+* **Example API Proto File**: automate/components/automate-gateway/api/compliance/reporting/
+* **Meta Swagger**: automate/components/automate-chef-io/data/docs/api-static/01-meta-swagger.json
+* **Reporting Export**: automate/components/automate-chef-io/data/docs/api-static/01-reporting-export.swagger.json
+* **Description**: automate/components/automate-chef-io/data/docs/api-static/02-meta-description.yaml
+* **Tags**: automate/components/automate-chef-io/data/docs/api-static/03-tags.swagger.json
+* **Compiled Swagger**: automate/static/api-docs/all-apis.swagger.json
+
+### Tags and Tag Groups
+
+* Endpoints need to appear in the rendered documentation human-readable groupings. We use group and endpoint tags to create an organized and meaningful presentation for the Chef Automate API.
+* Endpoints must have a tag that is a member of a tag group in order to appear in the rendered documentation. Endpoints without tags that are members of a tag group will not show up.
+* An endpoint's default tag is the parent service where it was defined.
+* Every service must have a unique tag name. For services without unique names, follow the explicit tagging process. Without unique service tag names, all endpoints for all of these services appear together in the rendered documentation.
+
+#### Automatic tagging
+
+Set up automatic tagging in the `components/automate-chef-io/data/docs/api-static/03-tags.swagger.json` file.
+
+* Add the service's name to the `tags` list in a `tagGroup` or create a new `tagGroup` with the service's name in the `tags` list.
+* Make the service name human-friendly by adding an entry to the `tags` in with a custom `x-displayName`.
+
+#### Explicit tagging
+
+* Each endpoint needs a tag specified in its definition, by adding a block like this to the endpoint definition in the proto file:
+
+    ```go
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: "My Custom Tag"
+    };
+    ```
+
+* All of the tags for an individual service should be the same, unless multiple groupings are desired in which case multiple independent tags can be used.
+* After this the process described for automatic tagging can be used, substituting the custom tag defined in place of the service's name.
+
+## Notes
+
+* Proto files can use tabs and spaces. We have a defacto convention for using tabs since our protos are primarily used with Go code.
+* Endpoints will not show up in the UI unless one or more of their tags are assigned to a tag group, but they will still appear in the Swagger file accessible through the docs in the browser.
+  * To explicitly hide an endpoint and remove it from the Swagger file, assign it the tag of `hidden` and it will be filtered out in pre-processing.
```

Comment:
```suggestion
* To explicitly hide an endpoint and remove it from the Swagger file, assign it the tag of `hidden` and it will be filtered out in pre-processing.
```

Reply:
If you meant to have this line as a sub-bullet, then ignore this suggestion! :)